inc/TRestRawBaseLineCorrectionProcess.h

```diff
@@ -75,5 +101,10 @@ class TRestRawBaseLineCorrectionProcess : public TRestEventProcess {
     // ROOT class definition helper. Increase the number in it every time
     // you add/rename/remove the process parameters
     ClassDefOverride(TRestRawBaseLineCorrectionProcess, 1);
```

Comment:
```suggestion
    ClassDefOverride(TRestRawBaseLineCorrectionProcess, 2);
```

When you change some of the members of the class (that are going to be serialized) you need to increase the class version (you changed `fSmoothingWindow` definition).